relay-server/src/utils/dynamic_sampling.rs

```diff
@@ -99,6 +215,75 @@ pub fn get_sampling_key(envelope: &Envelope) -> Option<ProjectKey> {
     envelope.dsc().map(|dsc| dsc.public_key)
 }
 
+/// Compares two semantic versions.
+///
+/// This function is just temporary since it's used to compare SDK versions when an [`Event`] is
+/// received.
+fn compare_versions(version1: &str, version2: &str) -> std::cmp::Ordering {
```

Comment:
Can we use the https://docs.rs/semver/latest/semver/ crate for this? It's already part of our dependency tree.

Reply:
I imagined there was something, will use that, thanks!